subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/ui/SubscriptionWebViewViewModel.kt

```diff
@@ -309,8 +309,8 @@ class SubscriptionWebViewViewModel @Inject constructor(
     ): SubscriptionOptionsJson {
         return SubscriptionOptionsJson(
             options = listOf(
-                createOptionsJson(yearlyOffer, YEARLY),
-                createOptionsJson(monthlyOffer, MONTHLY),
+                createOptionsJson(yearlyOffer, YEARLY.lowercase()),
+                createOptionsJson(monthlyOffer, MONTHLY.lowercase()),
             ),
```

Comment:
@lmac012 I had this commit locally, that's why you saw the regression in the paywall screen. Now, it should work as expected